plugin/builtin/helmchartinflationgenerator/HelmChartInflationGenerator_test.go

```diff
@@ -579,3 +581,201 @@ valuesInline:
 `)
 	th.AssertActualEqualsExpected(rm, "")
 }
+
+func TestHelmChartInflationGeneratorWithSameChartMultipleVersions(t *testing.T) {
+	th := kusttest_test.MakeEnhancedHarnessWithTmpRoot(t).
+		PrepBuiltin("HelmChartInflationGenerator")
+	defer th.Reset()
+	if err := th.ErrIfNoHelm(); err != nil {
+		t.Skip("skipping: " + err.Error())
+	}
+
+	tests := []struct {
+		name        string
+		chartName   string
+		repo        string
+		version     string
+		releaseName string
+	}{
+		{
+			name:        "terraform chart with no version grabs latest",
+			chartName:   "terraform",
+			repo:        "https://helm.releases.hashicorp.com",
+			version:     "",
+			releaseName: "terraform-latest",
+		},
+		{
+			name:        "terraform chart with version 1.1.1",
+			chartName:   "terraform",
+			repo:        "https://helm.releases.hashicorp.com",
+			version:     "1.1.1",
+			releaseName: "terraform-1.1.1",
+		},
+		{
+			name:        "terraform chart with version 1.1.1 again",
```

Comment:
I think the testcase name could be more specify than 'again'. and can you explain the difference between releaseName  in ``1.1.1``  and ``1.1.1-1`` ? ( I lack of context here)

Reply:
sorry, I might not have context as well, because this code I brought from a previous PR. But it seems to me that the test case is trying to ensure that using the same version of the chart will refer to the same directory as the previous case.

Reply:
any idea for the name? @charles-chenzz

Reply:
I think we need to figure out what the testcase means and then we can name it